### PR TITLE
Fix for new pods stuck in ContainerCreating state due to missing CM volume mount

### PR DESCRIPTION
**Pull request by aavarghese**
Signed-off-by: aavarghese <avarghese@us.ibm.com>

Part of #1537 

## Proposed Changes

- StatefulSet pod management policy changed to "Parallel" so subsequent pod replicas don't wait for previous pod to be in "Running" state

### CODE REVIEW

File: data-plane/config/sourcev2/500-dispatcher.yaml [hunk 1]
```diff
@@ -154,4 +156,4 @@ spec:
       restartPolicy: Always
       dnsConfig:
         options:
-          - name: single-request-reopen
+          - name: single-request-reopen
```
[the remark this answers is not in the document]
Comment (aavarghese) [reply data-plane/config/sourcev2/500-dispatcher.yaml:159]: Ooo yes - thank you!!!

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Comment by pierDipi** [comment]
Thanks!

**Pull request merged**